src/app/detail/ui/Sidebar.tsx

```diff
@@ -215,15 +215,15 @@ export const Sidebar = ({
 }
 
 export const SidebarSkeleton = () => {
-  const matches = useMediaQuery('(max-width:600px)')
+  const matches = useMediaQuery('(max-width:700px)')
 
   return (
     <Box
       sx={{
         borderLeft: (theme) => `1px solid ${theme.color.borders.border2}`,
         height: '100vh',
         display: 'block',
-        width: matches ? '100vw' : '25vw',
+        width: matches ? '100vw' : { sm: '16vw', md: '20vw' },
```

Comment:
I don't think we need to give varying height to solve this issue. Why don't we calculate the height of the title input box, and then calculate the height of the tapwrite description with respect to the height of the title input box? This makes the solution straightforward IMO.